### ISSUE TITLE
Heap reported by clinic is very different from that reported by node

### ISSUE THREAD

**Issue by llimllib** [clinicjs/node-clinic#458]
## Expected Behavior

- When I run a test and jest reports that it's using 308 mb of memory, I expect clinic to report that I was using 308 mb of memory

## Current Behavior

- Clinic is off by an order of magnitude

## Steps to Reproduce (for bugs)

1. Run jest under clinic with `--logHeapUsage`: `clinic doctor -- node node_modules/.bin/jest --logHeapUsage packages/models/__tests__/project`
2. Note that jest reports 469 mb max heap usage
  - `PASS   models  packages/models/__tests__/project/model.test.ts (10.117 s, 469 MB heap size)`
3. Note that clinic reports we used ~40:
<img width="700" alt="image" src="https://github.com/clinicjs/node-clinic/assets/7150/a1a4ea83-eea6-432e-9e80-f796742b9ece">

What could be causing such a large difference? [Jest uses](https://github.com/jestjs/jest/blob/7ed802537349ca259e83938d79f2d095d274d576/packages/jest-runner/src/runTest.ts#L355-L360) `process.memoryUsage().heapUsed` to measure the heap size.

## Sample upload

I don't feel comfortable uploading my work app's trace, unfortunately

## Environment

* Clinic.js version: v13.0.0 Doctor v11.0.0

* Node.js version: v16.20.0 npm 8.19.4

* Browser name and version: N/A

* Operating system and version: macOS Ventura 13.4

**Comment by llimllib** [comment]
I should have mentioned when I wrote this that `--runInBand` doesn't change the situation; I suspect it's something to do with the fact that node kicks off a child process. Here's the process tree, as reported by `htop` during a test run:

<img width="1761" alt="image" src="https://github.com/clinicjs/node-clinic/assets/7150/d7d2449f-16e8-478a-9d52-bcd9785d56d4">

You can see there that the main jest process only has 86mb, while its child test-runner process is chomping up an order of magnitude more memory.

Is there any way to get at the child process?

**Comment by RafaelGSS** [comment]
Not really. Heapprofiler measures only the main process, in your case the main jest process.
